share/ruby-build/truffleruby+graalvm-dev

```diff
@@ -1,18 +1,18 @@
 platform="$(uname -s)-$(uname -m)"
 case $platform in
 Linux-x86_64)
-  install_package "truffleruby+graalvm-dev" "https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest/download/graalvm-community-java21-linux-amd64-dev.tar.gz" truffleruby_graalvm
+  install_package "truffleruby+graalvm-dev" "https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest/download/truffleruby-community-java-dev-linux-amd64.tar.gz" truffleruby
```

Comment:
These URLs will soon switch from `-java-` to `-jvm-`, I'll make another PR to update that.